ulog/log.go

```diff
@@ -140,11 +140,13 @@ func Logger(ctx context.Context) Log {
 }
 
 // NewLogContext sets the context with the context aware logger
-func NewLogContext(ctx context.Context) context.Context {
+func NewLogContext(ctx context.Context, log Log) context.Context {
```

Comment:
This is used when middleware sets up Log object on the context with module specific fields.